dropwizard-jetty/src/main/java/io/dropwizard/jetty/HttpConnectorFactory.java

```diff
@@ -258,11 +258,11 @@ public static ConnectorFactory admin() {
     @MinSize(value = 1, unit = SizeUnit.BYTES)
     private Size maxBufferPoolSize = Size.kilobytes(64);
 
-    @Min(1)
-    private int acceptorThreads = Math.max(1, Runtime.getRuntime().availableProcessors() / 2);
+    @Min(-1)
+    private int acceptorThreads = -1;
 
-    @Min(1)
-    private int selectorThreads = Runtime.getRuntime().availableProcessors();
+    @Min(0)
+    private int selectorThreads = 0;
```

Comment:
I like the explicitness of `Optional` instead of people needing to know that `-1` has a special meaning.